lang/nl/strings.php

```diff
@@ -3,16 +3,34 @@
 return [
     'email' => 'E-mail',
     'email_address' => 'E-mailadres',
-    'user_identifier' => 'Gebruikersnaam of e-mail',
+    'user_identifier' => 'Gebruikersnaam of E-mail',
     'password' => 'Wachtwoord',
+    'password_confirmation' => 'Wachtwoordbevestiging',
+    'current_password' => 'Huidig Wachtwoord',
     'new_password' => 'Nieuw wachtwoord',
     'confirm_password' => 'Bevestig nieuw wachtwoord',
     'login' => 'Inloggen',
-    'home' => 'Startpagina',
-    'servers' => 'Servers',
+    'username' => 'Gebruikersnaam',
+    'first_name' => 'Voornaam',
+    'last_name' => 'Achternaam',
+    'language' => 'Taal',
+    'dashboard' => 'Dashboard',
     'id' => 'ID',
+    'uuid' => 'Uuid',
     'name' => 'Naam',
-    'node' => 'Node',
+    'description' => 'Beschrijving',
+    'api_keys' => 'API Sleutel|API Sleutels',
+    'databases' => 'Database|Databases',
+    'mounts' => 'Mount|Mounts',
+    'eggs' => 'Egg|Eggs',
+    'nodes' => 'Node|Nodes',
+    'servers' => 'Server|Servers',
+    'users' => 'Gebruiker|Gebruikers',
+    'force_outgoing_ip' => 'Forceer Uitgaande IP',
+    'startup' => 'Startup',
+    'docker_images' => 'Docker Images',
+    'linked_node' => 'Gekoppelde Node',
+    'primary_allocation' => 'Primaire Allocatie',
```

Comment:
Allocatie -> Toewijzing to be consistent with current lang files

Reply:
I haven't looked at the current lang file, since it wasn't in my folder when I started working on this PR